src/main/scala/io/github/sentenza/hacktoberfest18/algos/MutableSorting.scala

```diff
@@ -185,10 +188,44 @@ package io.github.sentenza.hacktoberfest18.algos
       array
     }
 
-    def bucketSort(xs: Array[Int]): Array[Int] = ???
+    def bucketSort(xs: Array[Int]): Array[Int] = bucketSort(xs, sqrt(xs.length).toInt, insertionSort)
```

Comment:
If you want you can also change the signature of this function here and in the trait itself. What is more, overloading this function in the same class/object seems a little bit odd in my opinion.

Here, you pass the `sqrt` and in the body of the function you do the same. Why doing that twice?

Reply:
No you're right I forgot I had already defined it in the body of the method. I'll remove that